PokeAlarm/Manager.py

```diff
@@ -373,7 +364,7 @@ def process_pokemon(self, pkmn):
                 log.debug("Pokemon 'cp' was not checked because it was missing.")
 
             # Check the Level of the Pokemon
-            if level != '?':
+            if level is not None and level != 'unkn' and level != '?':
```

Comment:
Why do we want to move to checking for these three things? level should never be None at this point